### PR TITLE
Support for node-moonbeam

### PR DESCRIPTION
**Pull request by jacogr**
Part of https://github.com/polkadot-js/apps/issues/3519

### CODE REVIEW

File: packages/page-staking/src/useSortedTargets.ts [hunk 1]
```diff
@@ -64,18 +64,20 @@ function sortValidators (list: ValidatorInfo[]): ValidatorInfo[] {
 }
 
 function extractSingle (allAccounts: string[], amount: BN = baseBalance(), { info }: DeriveStakingElected | DeriveStakingWaiting, favorites: string[], perValidatorReward: BN, isElected: boolean): [ValidatorInfo[], string[], BN, BN] {
-  const nominators: string[] = [];
+  const defaultExposure = {
```
Comment (jacogr) [inline packages/page-staking/src/useSortedTargets.ts:67]: Erm, was not meant to be includes here, finger error - extracted this specific on in https://github.com/polkadot-js/apps/pull/3529 (innocent, but really doesn't belong here, part of a larger PR elsewhere)

**Comment by crystalin** [comment]
Thank you for taking the initiative on this Jaco, that's really nice of you

**Comment by joelamouche** [comment]
@jacogr Let's coordinate on this PR as soon as you are free. I sent you a message on disc

**Comment by jacogr** [comment]
@joelamouche Saw the PR, that is absolutely perfect. As it stands this one has a lot of holes and points to _some_ stuff, but there has also been changes since. so very outdated.

**Comment by joelamouche** [comment]
@jacogr Great! I would love to sync on what the remaining holes are.
How do you recommend I sync my branch with what has been developed since?
Let's coordinate on element if that works for you or through email if you could provide me one.

Thanks in advance,

Antoine

**Comment by jacogr** [comment]
Closing in favor of https://github.com/polkadot-js/apps/pull/3961

**Pull request closed without merging**